CMakeLists.txt

```diff
@@ -56,19 +56,3 @@ else()
 	add_subdirectory(3rdparty/hidapi/libusb)
 	#list(APPEND LIBS hidapi-libusb)
 endif()
-
-# Linux installation
```

Comment:
- `install(TARGETS ...)` only works after a target is defined e.g., `add_executable()`
- because `install(FILES ...)` doesn't strip binaries
- Trimming unnecessary indirection and grouping `install` commands to be relative to `project()`